bindings/java/src/main/com/apple/foundationdb/NativeFuture.java

```diff
@@ -44,6 +44,10 @@ protected NativeFuture(long cPtr) {
 	//
 	// Since this must be called from a constructor, we assume that close
 	// cannot be called concurrently.
+	//
+	// <p>
+	//    Note: This function guarantees the callback will be executed **at most once**.
+	// </p>
```

Comment:
Since this isn't javadoc and I believe this function isn't part of our public API, I think we can omit the formatting tags.

Reply:
Done